From 2b8d89177608810e086b9db40c097b44657a847f Mon Sep 17 00:00:00 2001
From: TuanTNT19 <tuantnt08@gmail.com>
Date: Sun, 5 Apr 2026 10:37:55 +0700
Subject: [PATCH] netsink.c: fix connect func

---
 lib/src/phy/io/netsink.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/src/phy/io/netsink.c b/lib/src/phy/io/netsink.c
index 1cc599b41..514f7866f 100644
--- a/lib/src/phy/io/netsink.c
+++ b/lib/src/phy/io/netsink.c
@@ -84,7 +84,7 @@ int srsran_netsink_set_nonblocking(srsran_netsink_t* q)
 int srsran_netsink_write(srsran_netsink_t* q, void* buffer, int nof_bytes)
 {
   if (!q->connected) {
-    if (connect(q->sockfd, &q->servaddr, sizeof(q->servaddr)) < 0) {
+    if (connect(q->sockfd, (const struct sockaddr*)&q->servaddr, sizeof(q->servaddr)) < 0) {
       if (errno == ECONNREFUSED || errno == EINPROGRESS) {
         return SRSRAN_SUCCESS;
       } else {
-- 
2.25.1

